app/notification/routes.py

```diff
@@ -28,14 +29,16 @@ def send_notification() -> dict:
     """
     notification_data = request.get_json()
     data = Notification.process_notification_data(notification_data)
-
-    response = notifications_client.send_email_notification(
-        email_address=data.contact_info,
-        template_id=TEMPLATE_ID,
-        personalisation={
-            "SUBJECT": "Funding service access link",
-            "MAGIC_LINK": data.content,
-        },
-    )
-
-    return response
+    if data:
+        response = notifications_client.send_email_notification(
+            email_address=data.contact_info,
+            template_id=TEMPLATE_ID,
+            personalisation={
+                "SUBJECT": "Funding service access link",
+                "MAGIC_LINK": data.content,
+            },
+        )
+
+        return response
+    else:
+        abort(400)
```

Comment:
🤔 Add a description to this abort if possible - and think about if a 400 error code is always correct in this case :)

Reply:
in most cases, if the data is received in & in the correct format then the notification should work fine so I could only think of 400? Will look into more detail